server/src/main/java/org/elasticsearch/action/admin/indices/analyze/DetailAnalyzeResponse.java

```diff
@@ -353,13 +325,10 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
-    public static class CharFilteredText implements Streamable, ToXContentObject {
+    public static class CharFilteredText implements Writeable, ToXContentObject {
```

Comment:
should we make class variables `name` and `texts`  `final` ?

Reply:
++